sliver_app_bar_example/android/app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.sliverappbar.sliver_app_bar_example"
+        applicationId "github.nisrulz.sliver_app_bar_example"
```

Comment:
The package name doesn't just change in one place. I only highlighted one instance. You will have to do a refactoring task to change it everywhere.

Reply:
Ya , thanks for the review again , and sorry for the silly mistakes :) 
Updated the package name again , should be good to merge.